### DISCUSSION TITLE
Virtual Pin Problem

### DISCUSSION THREAD

**Discussion by Maddin4000** [KlipperScreen/KlipperScreen#1342] [Q&A]
hi everyone, what can I do?
<img width="560" alt="Bildschirmfoto 2024-04-21 um 22 39 23" src="https://github.com/KlipperScreen/KlipperScreen/assets/115088579/1875fe93-0571-4a93-83c5-33e511d03a48">

**Comment by alfrix** [comment]
This is unrelated to KlipperScreen, please ask on the klipper discord or discourse
